testing/testcafe/tests/navigation/contextMenu/common.ts

```diff
@@ -1,49 +1,62 @@
 import { Selector } from 'testcafe';
```

Comment:
What's the difference between `contextMenu/common.ts` and `contextMenu/contextMenu.ts`? Why not the single united file?

Reply:
We should use common.ts for general tests on the component. The alias name `widgetName.ts` used earlier. We may refactor it in future merges.